doc/OnlineDocs/contributed_packages/incidence/tutorial.dm.rst

```diff
@@ -53,8 +53,7 @@ with fields for each of the four subsets defined by the partition:
 If any variables or constraints are unmatched, the (Jacobian of the) model
 is structurally singular.
 
-.. doctest::
-   :skipif: not scipy_available or not networkx_available or not asl_available
+.. code-block:: python
```

Comment:
Can we leave a comment as to why this is not being tested?  Or better, why is this nondeterministic?

Reply:
Added a comment noting that the unmatched variables/constraints are not mathematically unique, and expanded on this in the text, noting that they depend on (a) how we identify incident variables and (b) the implementation of the matching algorithm.